doc/engineering-reference/src/climate-sky-and-solar-shading-calculations/shading-module.tex

```diff
@@ -406,39 +406,54 @@ \subsubsection{Overlapping Shadows}\label{overlapping-shadows}
 \caption{Complex Overlapping Condition \protect \label{fig:complex-overlapping-condition}}
 \end{figure}
 
-If two shadows overlap the receiving surface, they may also overlap each other as in Figure~\ref{fig:multiple-shadow-overlaps}. The vertices of this overlap can be computed.~ The areas of all overlaps can be computed.~ The total sunlit area can be expressed as the sum of all polygon areas given a proper sign on each of the areas.
+If two shadows overlap the receiving surface, they may also overlap each other as in Figure~\ref{fig:multiple-shadow-overlaps}. The vertices of this overlap can be computed, and the areas of all overlaps can be computed. For opaque shadows, the total sunlit area can be expressed as the sum of all polygon areas using the sign convention shown in Table \ref{table:surface-area-characteristic-convention}:
 
-The following convention was adopted:
+\begin{equation}
+SunlitArea = {\sum\limits_{i = 1}^n {A_i}}
+\end{equation}
 
-% table 24
-\begin{longtable}[c]{@{}ll@{}}
-\caption{Surface / Area Characteristic / Convention \label{table:surface-area-characteristic-convention}} \tabularnewline
+Each shadow's area is subtracted from the receiving surface and so on through multiple overlaps where the sign of the overlap area is the product of the signs of the overlapping areas. For the shadows in Figure~\ref{fig:multiple-shadow-overlaps}, start with the receiving surface area A, subtract shadow area B, then subtract shadow area C. Because shadows B and C overlap, the overlap area D has been subtracted twice, so add back area D to get the final sunlit area on the receiving surface.
+
```

Comment:
I understand the math now. The key is the sign convention when adding in the area * transmittance values. The 2 shadows "area * trans" are substracted from the total area. That winds up being a (1 - HCT_1 + HCT_2) term for the overlap. Then the overlap is added back to that value using the difference of full trans - actual trans which gets back to actual trans as (1 - HCT_1 + HCT_2) + (HCT_1 + HCT_2) - (HCT_1 * HCT_2) for the overlap. This yields (1 - (HCT_1 * HCT_2)) as the sunlit fraction for the overlap area. This is the original equation in the model. But that is not what needs to get added back as @mjwitte has shown. The overlap area needs to add back (HCT_1 + HCT_2) - (HCT_1 * HCT_2) to get back to the correct value which @mjwitte has said multiple times. This model therefore calculates the resulting sunlit area fraction based on shading surface transmittances.

Reply:
As another test, I used 2 shading surfaces with 0.7 transmittance that covered the entire area and had a miniscule overlap. I would expect a sunlit area fraction of 0.7. The spreadsheet shows 0.699998.

![image](https://github.com/NREL/EnergyPlus/assets/5778156/93088607-a4bd-4e9c-9a08-d1ccd21add4e)

Reply:
I did make sure to check the other extreme here with overlapped shades with area 19.9999 with miniscule NON overlap (D = 19.9998) and expected a result of 0.49. The actual result was 0.490002.